src/Scripting/CSharpTest/CommandLineRunnerTests.cs

```diff
@@ -135,7 +135,7 @@ public void Await()
                 runner.Console.Error.ToString());
         }
 
-        [Fact]
+        [Fact(Skip="https://github.com/dotnet/roslyn/issues/17043")]
```

Comment:
Oh ok, this test is failing because inside of a scripting context, it is setting and reading DefaultThreadCurrentUICulture which is an AsyncLocal property.

Due to how the scripting engine uses Tasks, this makes any write invisible.

> why are we disabling it instead of fixing it or limiting it to environments on which it is passing

The fact that this test passes on English machines is a seperate bug that I don't understand yet.  I've disabled it so that I can try to find out why it passes on English machines.

> BTW, what is CSI in this context?

I just meant "scripting engine"